src/components/user/HomeMap.js

```diff
@@ -1,10 +1,31 @@
-import React from 'react';
-
-import { MapContainer, TileLayer, Marker, Popup } from '@monsonjeremy/react-leaflet';
+import React, { useEffect, useState } from 'react';
+import { MapContainer, TileLayer, Marker, Popup, useMap } from '@monsonjeremy/react-leaflet';
 import 'leaflet/dist/leaflet.css';
 
-const HomeMap = () => {
-  const position = [-34.907, -56.203];
+const DEFAULT_LONGITUDE = -34.907;
+const DEFAULT_LATITUDE = -56.203;
+
+const position = [DEFAULT_LATITUDE, DEFAULT_LONGITUDE];
+
+function HomeMap() {
+  function LocationMarker() {
+    const [position, setPosition] = useState(null);
```

Comment:
I don't think I'll set a default location; as the whole point of the app is based on the fact that it has access to your location. If the user doesn't grant permissions, the whole app becomes pointless and I don't think it's relevant to set a default location.

Reply:
I was referring to the initial location shown when the map loads, before permissions are granted. It was just a suggestion, since you had a `DEFAULT_LATITUDE` and `DEFAULT_LONGITUDE` constants unused.